serif_cli/src/main/kotlin/serif_cli/App.kt

```diff
@@ -29,19 +30,52 @@ class App {
             println()
             return
         }
+
+        //Each UI will create it's specific DriverFactory
+        //And call this function before the backend can get
+        //information out of the database
+        Database.initDb(DriverFactory())
```

Comment:
I think the shared code can create the driver itself, right? It feels a bit out of place in the UI.

Reply:
You would be mostly right, unfortunately the android driver requires a Context var to be passed in, so we would need to pass the android context to the backend.

Reply:
gah that's a good point. Android man